src/main/resources/application.yaml

```diff
@@ -9,3 +9,11 @@ management:
 spring:
   application:
     name: SscsBulkScan
+
+ccd:
+ case:
+  jurisdiction: ${CCD_CASE_JURISDICTION:SSCS}
```

Comment:
can you drop the env vars please? these map exactly to what spring would convert